src/core/src/op/divide.cpp

```diff
@@ -96,109 +82,108 @@ bool evaluate_bound(const Node* node, ov::TensorVector& output_values, bool is_u
     OPENVINO_ASSERT(PartialShape::broadcast_merge_into(input_shape, input2.get_partial_shape(), node->get_autob()),
                     "Argument shapes in divide operation are inconsistent.");
 
-    auto input1_low = ov::evaluate_lower_bound(input1);
+    const auto input1_low = evaluate_lower_bound(input1);
     if (!input1_low)
         return false;
-    auto input1_up = ov::evaluate_upper_bound(input1);
+    const auto input1_up = evaluate_upper_bound(input1);
     if (!input1_up)
         return false;
-    auto input2_low = ov::evaluate_lower_bound(input2);
+    const auto input2_low = evaluate_lower_bound(input2);
     if (!input2_low)
         return false;
-    auto input2_up = ov::evaluate_upper_bound(input2);
+    const auto input2_up = evaluate_upper_bound(input2);
     if (!input2_up)
         return false;
 
-    auto zeros_const = op::Constant::create(input2.get_element_type(), {}, {0});
-    const auto zero_t = ov::Tensor(input2.get_element_type(), Shape{});
+    const auto zeros_const = Constant::create(input2.get_element_type(), {}, {0});
+    const auto zero_t = Tensor(input2.get_element_type(), Shape{});
     memcpy(zero_t.data(), zeros_const->get_data_ptr(), zero_t.get_byte_size());
 
     OPENVINO_SUPPRESS_DEPRECATED_START
-    auto max_constant = get_constant_max_of_type(input2.get_element_type());
-    auto dynamic_mask = or_tensor(equality_mask(input1_up, max_constant), equality_mask(input2_up, max_constant));
+    const auto max_constant = ngraph::get_constant_max_of_type(input2.get_element_type());
```

Comment:
```suggestion
    const auto max_constant = ov::util::make_tensor_of_max_value(input2.get_element_type());
```
Consider use this version, but then function `equality_mask` should be refactored to use tensors as parameters instead tensor and Constant (will also simpler as there will be no copy of const value into new local tensor).

If used then `OPENVINO_SUPPRESS_DEPRECATED_START` could be removed.

Reply:
Done